libs/MVS/DepthMap.cpp

```diff
@@ -2107,6 +2107,28 @@ bool MVS::ImportDepthDataRaw(const String& fileName, String& imageFileName,
 } // ImportDepthDataRaw
 /*----------------------------------------------------------------*/
 
+bool MVS::GetDepthMapHeaderSize(const String& fileName, cv::Size& size)
```

Comment:
I think this was a interesting exercise, but function `ImportDepthDataRaw()` already existing could be already used for this if you set `flags` param to 0, pls check

Reply:
if you want you can still let this function, but instead of duplicating the code just call `ImportDepthDataRaw` and return the only the sizes;

more useful will be to rename it to `std::optional<std::pair<cv::Size, cv::Size>> GetDepthMapHeaderSizes(const String& fileName)`